system-contracts/contracts/L2BaseToken.sol

```diff
@@ -67,7 +67,7 @@ contract L2BaseToken is IBaseToken, ISystemContract {
         emit Mint(_account, _amount);
     }
 
-    /// @notice Initiate the ETH withdrawal, funds will be available to claim on L1 `finalizeEthWithdrawal` method.
+    /// @notice Initiate the withdrawal of the base token, funds will be available to claim on L1 `finalizeEthWithdrawal` method.
```

Comment:
The `ETH` here has been changed to `base token` and that makes sense. Should we also consider changing the name of `finalizeEthWithdrawal` since it's no longer necessarily ETH?

Reply:
Unfortunately no, since we need the old SDKs to be backward-compatible, this function is also only available on Era